src/app/ApplicationTemplate.Droid/ApplicationTemplate.Droid.csproj

```diff
@@ -47,6 +47,8 @@
 		<DefineConstants>DEBUG;TRACE</DefineConstants>
 		<Optimize>False</Optimize>
 		<OutputPath>bin\Debug\</OutputPath>
+		<AndroidUseSharedRuntime>True</AndroidUseSharedRuntime>
```

Comment:
Est-ce qu'il faudrait pas le changer en release aussi (même si c'est le default? L'enlever du default et le mettre en release pour la clarté)

Reply:
From my comprehension of [this page](https://docs.microsoft.com/en-us/xamarin/android/deploy-test/building-apps/build-process), this property only concerns debug builds.